rpc/http.go

```diff
@@ -245,6 +246,12 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if origin := r.Header.Get("Origin"); origin != "" {
 		ctx = context.WithValue(ctx, "Origin", origin)
 	}
+	if s.debugSingleRequest {
+		if v := r.Header.Get(dbg.HTTPHeader); v == "true" {
```

Comment:
how hard it is to move -H "dbg: true" into the JSON body? it used to be that some web clients could not send custom HTTP headers

Reply:
but it may be useful: because some clients are connected by `websocket` and sending many requests per 1 connections

Reply:
instead of parsing consider just looking for a "dbg: true" substring in the request bytes. could be good enough

Reply:
`r.Header` is `type Header map[string][]string` means it's already parsed there

Reply:
Sure, I mean that if we want to use "dbg: true" inside JSON, we don't have to parse the request body to JSON and add parameters to API funcs. Just looking for a "dbg: true" substring in the request bytes might be good enough.

It used to be that some web APIs, and some firewalls/proxies don't allow custom headers.

Is it possible to send a custom HTTP header from a diagnostics web page, @dvovk ?

Reply:
@battlmonstr I don't have such functionality now. But it is pretty easy to add it.